Tools/MonoGame.Effect.Compiler/Effect/TPGParser/PassInfo.cs

```diff
@@ -1,9 +1,13 @@
-﻿using System;
+﻿// MonoGame - Copyright (C) MonoGame Foundation, Inc
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+using System;
 using System.Text.RegularExpressions;
 using Microsoft.Xna.Framework.Graphics;
 using System.Globalization;
 
-namespace MonoGame.Effect.TPGParser
+namespace MonoGame.Effect
```

Comment:
@harry-cpp i moved these directly to the `Effect` namespace since they need to be public for `ShaderProfile` to work.

Reply:
Should i move them out one folder too?

Reply:
It's an internal method, we just need to mark all the classes as internal instead of public and it should work fine without expositing the methods no?